source/uninstalling.rst

```diff
@@ -0,0 +1,70 @@
+.. _pdp.uninstalling:
+
+Uninstalling |pdp|
+********************************************************************************
+
+To uninstall |pdp|, remove all the installed packages and data / configuration files. 
+
+.. note::
+
+   Should you need the data files later, back up your data before uninstalling |pdp|.
+
+
+Using the |deb| format
+================================================================================
+  
+To uninstall |pdp| on platforms that use |apt| package manager such as Debian
+or Ubuntu, complete the following steps.
+
+Run all commands as root or via |sudo|.
+
+1. Stop the |pdp| service.
+
+   .. code-block:: bash
+
+      $ sudo systemctl start postgresql.service
+
+#. Remove the |percona-postgresql| packages
+
+   .. code-block:: bash
+
+      $ sudo apt-get remove percona-postgresql-12* percona-pgbackrest percona-patroni
+ 
+#. Remove configuration files         
+
+   .. code-block:: bash
+
+      $ rm -rf /etc/postgresql/12
```

Comment:
Not only configuration files, data will be deleted too.

Reply:
:+1: